solutions/cses-2428.mdx

```diff
@@ -74,4 +74,60 @@ public class cses2428 {
 
 </JavaSection>
 
+<CPPSection>
+
+```cpp
+/*
+Pranav Chatur
+https://github.com/DarkMenacer
+12 April 2022
+*/
+
+#include <bits/stdc++.h>
+#define ll long long
+#define vll vector<ll>
+using namespace std;
+
+ll give_me_answer(vll &integers, ll k){
+    ll answer = 0;
+    ll i = 0, j = 1;
+    map<ll,ll> distinct;
+    distinct[integers[i]]++;
+    while(1){
+        if(j==integers.size()+1){
+	/*
+	When j reaches end of the integers, there is no need to subtract 'total subarrays from i+1 to j',
+	(as no new element is going to be added further), therefore answer+= (j-i)*(j-i-1)/2
+	*/
+            answer += (j-i)*(j-i-1)/2;
```

Comment:
if you're going to use a space here, make it consistent between things like +-*/